src/main/java/appeng/me/cluster/implementations/CraftingCPUCluster.java

```diff
@@ -1587,6 +1601,36 @@ public int getRemainingOperations() {
         }
     }
 
+    public void tryExtractItems() {
+        if (!isMissingMode || this.waitingForMissing.isEmpty()) return;
+        if (countToTryExtractItems > 1200) {
+            countToTryExtractItems = 0;
+            for (IAEItemStack waitingForItem : this.waitingForMissing) {
+                final IGrid grid = this.getGrid();
+                if (grid != null) {
+                    final IStorageGrid pg = grid.getCache(IStorageGrid.class);
+                    if (pg != null) {
+                        IAEItemStack extractedItems = pg.getItemInventory()
+                                .extractItems(waitingForItem, Actionable.MODULATE, this.machineSrc);
+                        if (extractedItems != null) {
+                            IAEStack notInjected = injectItems(extractedItems, Actionable.MODULATE, this.machineSrc);
+                            if (notInjected != null) { // not sure if this even need, but still
+                                for (String playerName : MinecraftServer.getServer().getAllUsernames()) {
+                                    getPlayerByName(playerName).addChatMessage(
```

Comment:
Console log is probably fine here, though I appreciate the funny message 😄

Reply:
alright, done